Mage.Sets/src/mage/cards/i/IcebreakerKraken.java

```diff
@@ -80,9 +82,15 @@ public IcebreakerKraken copy() {
 
 class IcebreakerKrakenEffect extends OneShotEffect {
 
+    private static final FilterPermanent filter = new FilterPermanent();
+
+    static {
+        filter.add(Predicates.or(CardType.CREATURE.getPredicate(), CardType.ARTIFACT.getPredicate()));
```

Comment:
The filter text isn't used in this effect, so `StaticFilters.FILTER_PERMANENT_ARTIFACT_OR_CREATURE` should be sufficient without making a custom filter